terraform/projects/app-whitehall-backend/main.tf

```
@@ -177,6 +177,20 @@ resource "aws_iam_role_policy_attachment" "whitehall_csvs_attach" {
   policy_arn = "${aws_iam_policy.s3_writer.arn}"
 }
 
+data "aws_security_group" "whitehall-rds" {
+  name = "${var.stackname}_whitehall_rds_access"
+}
+
+resource "aws_security_group_rule" "whitehall-rds_ingress_whitehall-backend_mysql" {
+  type      = "ingress"
+  from_port = 3306
+  to_port   = 3306
+  protocol  = "tcp"
+
+  security_group_id        = "${aws_security_group.signon-rds.id}"
```

This one looks like an oopsy
```suggestion
  security_group_id        = "${aws_security_group.whitehall-rds.id}"
```